packages/experiments/src/components/FloatingSuggestionsComposite/FloatingSuggestionsComposite.test.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import { create, act } from 'react-test-renderer';
+import { BaseFloatingSuggestions } from './FloatingSuggestions';
+import * as ReactDOM from 'react-dom';
+import { IFloatingSuggestionItem } from './FloatingSuggestionsItem/FloatingSuggestionsItem.types';
+import { IBaseFloatingSuggestionsProps } from './FloatingSuggestions.types';
+import * as ReactTestUtils from 'react-dom/test-utils';
+
+export interface ISimple {
+  key: string;
+  name: string;
+}
+
+let container: HTMLDivElement | null;
+let _suggestions: IFloatingSuggestionItem<ISimple>[];
+
+const items: ISimple[] = [
+  { key: '1', name: 'a' },
+  { key: '2', name: 'b' },
+];
+
+const _onSuggestionSelected = jest.fn();
+const _onSuggestionRemoved = jest.fn();
+
+const picker = React.createRef<HTMLDivElement>();
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+  _suggestions = ([
+    {
+      key: '1',
+      id: '1',
+      displayText: 'Suggestion 1',
+      item: items[0],
+      isSelected: false,
+      showRemoveButton: true,
+      name: 'Alexa',
+    },
+    {
+      key: '2',
+      id: '2',
+      displayText: 'Suggestion 2',
+      item: items[1],
+      isSelected: false,
+      showRemoveButton: true,
+      name: 'Alexis',
+    },
+  ] as unknown) as IFloatingSuggestionItem<ISimple>[];
+});
+
+afterEach(() => {
+  if (container) {
+    document.body.removeChild(container);
+    ReactDOM.unmountComponentAtNode(container);
+    container = null;
+  }
+  jest.clearAllMocks();
+});
+
+export type TypedFloatingSimpleSuggestionsProps = Omit<IBaseFloatingSuggestionsProps<ISimple>, 'onRenderSuggestion'>;
+
+describe('FloatingSuggestions', () => {
+  const renderNothing = () => <></>;
+  const isSuggestionsVisible = false;
+  it('renders FloatingSuggestions correctly', () => {
+    const component = create(
+      <BaseFloatingSuggestions
+        onRenderNoResultFound={renderNothing}
+        isSuggestionsVisible={isSuggestionsVisible}
+        suggestions={[]}
+        targetElement={null}
+      />,
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders FloatingSuggestions with suggestions visible false', () => {
+    const component = create(
+      <BaseFloatingSuggestions
+        isSuggestionsVisible={isSuggestionsVisible}
+        suggestions={_suggestions}
+        targetElement={null}
+      />,
+    );
+
+    expect(component).toBeTruthy();
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders FloatingSuggestions with suggestions visible true', () => {
+    // Our functional tests need to run against actual DOM for callouts to work,
+    // since callout mount a new react root with ReactDOM.
+    //
+    // see https://github.com/facebook/react/pull/12895
+    act(() => {
+      (ReactDOM.render(
```

Comment:
If you can, I'd suggest switching to enzyme using `mount` instead. That said, more tests are better than less tests so I'll approve and it's up to you if you want to swap it out are not.